src/containers/CharacterProfile/AttackList.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import {
+    View,
+    Text,
+    ListView,
+    ScrollView,
+    Button,
+    TextInput,
+    Platform,
+    Dimensions,
+    Animated
+} from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+
+//images
+import headshots from '../../img/headshots/index';
```

Comment:
We can probably change the exported object and the folder here to be CharacterImages to simplify stuff.